pkg/manifests/apiserver_config.go

```diff
@@ -35,24 +35,24 @@ func NewAPIServerConfig(config *configv1.APIServer) *APIServerConfig {
 	}
 }
 
-// GetTLSCiphers returns the TLS ciphers for the
+// TLSCiphers returns the TLS ciphers for the
 // TLS security profile defined in the APIServerConfig.
-func (c *APIServerConfig) GetTLSCiphers() []string {
+func (c *APIServerConfig) TLSCiphers() []string {
 	profile := c.getTLSProfile()
 	if len(profile.Ciphers) == 0 {
 		return APIServerDefaultTLSCiphers
 	}
 	return profile.Ciphers
 }
 
-// GetMinTLSVersion returns the minimum TLS version for the
+// MinTLSVersion returns the minimum TLS version for the
 // TLS security profile defined in the APIServerConfig.
-func (c *APIServerConfig) GetMinTLSVersion() configv1.TLSProtocolVersion {
+func (c *APIServerConfig) MinTLSVersion() string {
```

Comment:
Trying to understand why this type change (TLSProtocolVersion to string) is needed?

Reply:
not strictly needed but I felt that the `minTLSVersioner` interface didn't need to be coupled to the `github.com/openshift/api/config/v1` package.